server/registry/actions_artifacts_test.go

```diff
@@ -373,7 +373,7 @@ func TestCreateArtifactDuplicates(t *testing.T) {
 				ArtifactId: "My-Artifact",
 				Artifact:   &rpc.Artifact{},
 			},
-			want: codes.AlreadyExists,
+			want: codes.InvalidArgument,
```

Comment:
Here I guess that we're getting the InvalidArgument result now because the validation check is happening first. I had a similar thing pop up in a different PR (https://github.com/apigee/registry/pull/615#discussion_r902911965), and there @seaneganx asked me to just remove the test because validation was already being tested.

Reply:
sure